src/Features/ScreenSpaceShadows.h

```diff
@@ -40,14 +40,16 @@ struct ScreenSpaceShadows : Feature
 
 	struct alignas(16) RaymarchCB
 	{
-		DirectX::XMFLOAT2 BufferDim;
-		DirectX::XMFLOAT2 RcpBufferDim;
-		DirectX::XMMATRIX ProjMatrix;
-		DirectX::XMMATRIX InvProjMatrix;
-		DirectX::XMFLOAT4 DynamicRes;
-		DirectX::XMVECTOR InvDirLightDirectionVS;
+		float2 BufferDim;
+		float2 RcpBufferDim;
+		float4x4 ProjMatrix[2];
+		float4x4 InvProjMatrix[2];
+		float4 CameraData;
+		float4 DynamicRes;
+		float4 InvDirLightDirectionVS;
```

Comment:
Check for issues with flat since it's changing content buffer structure.

Reply:
Tested in flatrim thoroughly now, downloaded a modpack and compared to a version before I started, exactly same. Asked doodlum about the affected Flatrim code and he said it was fine to use CameraData for flatrim (even said it was more correct as well)